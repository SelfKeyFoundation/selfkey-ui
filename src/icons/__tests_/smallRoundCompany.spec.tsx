import * as React from 'react';
import { create } from 'react-test-renderer';
import SmallRoundCompany from '../round-company-small';

describe('Small Round Company Icon', () => {
  it('should show Small Round Company Icon', () => {
    const component = create(
      <SmallRoundCompany />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});