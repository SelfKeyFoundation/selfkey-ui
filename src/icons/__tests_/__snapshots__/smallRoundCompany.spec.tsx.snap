// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Small Round Company Icon should show Small Round Company Icon 1`] = `
<svg
  aria-hidden="true"
  className="MuiSvgIcon-root"
  focusable="false"
  style={
    Object {
      "height": "28px",
      "width": "28px",
    }
  }
  viewBox="0 0 28 28"
>
  <g
    fill="none"
    fillRule="evenodd"
    id="ðŸ–¥-Main-Dashboard"
    stroke="none"
    strokeWidth="1"
  >
    <g
      id="300.5-Navigation-open---switch-profile"
      transform="translate(-1196.000000, -143.000000)"
    >
      <g
        id="Top-Nav-Dropdown"
        transform="translate(1182.000000, 64.000000)"
      >
        <g
          id="Alias2"
          transform="translate(15.000000, 77.000000)"
        >
          <g
            id="icon-round-company"
            transform="translate(0.000000, 3.000000)"
          >
            <circle
              cx="13"
              cy="13"
              id="Oval-2"
              r="13"
              stroke="#697C95"
              strokeWidth="1.56521739"
            />
            <path
              d="M11.0138889,10.7712674 L11.0138889,9.81206597 C11.0138889,9.73213212 11.0418653,9.66418937 11.097819,9.60823568 C11.1537727,9.55228198 11.2217155,9.52430556 11.3016493,9.52430556 L12.2608507,9.52430556 C12.3407845,9.52430556 12.4087273,9.55228198 12.464681,9.60823568 C12.5206347,9.66418937 12.5486111,9.73213212 12.5486111,9.81206597 L12.5486111,10.7712674 C12.5486111,10.8512012 12.5206347,10.919144 12.464681,10.9750977 C12.4087273,11.0310514 12.3407845,11.0590278 12.2608507,11.0590278 L11.3016493,11.0590278 C11.2217155,11.0590278 11.1537727,11.0310514 11.097819,10.9750977 C11.0418653,10.919144 11.0138889,10.8512012 11.0138889,10.7712674 Z M14.3710938,11.0590278 C14.2911599,11.0590278 14.2232171,11.0310514 14.1672635,10.9750977 C14.1113098,10.919144 14.0833333,10.8512012 14.0833333,10.7712674 L14.0833333,9.81206597 C14.0833333,9.73213212 14.1113098,9.66418937 14.1672635,9.60823568 C14.2232171,9.55228198 14.2911599,9.52430556 14.3710938,9.52430556 L15.3302951,9.52430556 C15.410229,9.52430556 15.4781717,9.55228198 15.5341254,9.60823568 C15.5900791,9.66418937 15.6180556,9.73213212 15.6180556,9.81206597 L15.6180556,10.7712674 C15.6180556,10.8512012 15.5900791,10.919144 15.5341254,10.9750977 C15.4781717,11.0310514 15.410229,11.0590278 15.3302951,11.0590278 L14.3710938,11.0590278 Z M11.3016493,13.3611111 C11.2217155,13.3611111 11.1537727,13.3331347 11.097819,13.277181 C11.0418653,13.2212273 11.0138889,13.1532845 11.0138889,13.0733507 L11.0138889,12.1141493 C11.0138889,12.0342155 11.0418653,11.9662727 11.097819,11.910319 C11.1537727,11.8543653 11.2217155,11.8263889 11.3016493,11.8263889 L12.2608507,11.8263889 C12.3407845,11.8263889 12.4087273,11.8543653 12.464681,11.910319 C12.5206347,11.9662727 12.5486111,12.0342155 12.5486111,12.1141493 L12.5486111,13.0733507 C12.5486111,13.1532845 12.5206347,13.2212273 12.464681,13.277181 C12.4087273,13.3331347 12.3407845,13.3611111 12.2608507,13.3611111 L11.3016493,13.3611111 Z M14.3710938,13.3611111 C14.2911599,13.3611111 14.2232171,13.3331347 14.1672635,13.277181 C14.1113098,13.2212273 14.0833333,13.1532845 14.0833333,13.0733507 L14.0833333,12.1141493 C14.0833333,12.0342155 14.1113098,11.9662727 14.1672635,11.910319 C14.2232171,11.8543653 14.2911599,11.8263889 14.3710938,11.8263889 L15.3302951,11.8263889 C15.410229,11.8263889 15.4781717,11.8543653 15.5341254,11.910319 C15.5900791,11.9662727 15.6180556,12.0342155 15.6180556,12.1141493 L15.6180556,13.0733507 C15.6180556,13.1532845 15.5900791,13.2212273 15.5341254,13.277181 C15.4781717,13.3331347 15.410229,13.3611111 15.3302951,13.3611111 L14.3710938,13.3611111 Z M12.5486111,15.375434 C12.5486111,15.4553679 12.5206347,15.5233106 12.464681,15.5792643 C12.4087273,15.635218 12.3407845,15.6631944 12.2608507,15.6631944 L11.3016493,15.6631944 C11.2217155,15.6631944 11.1537727,15.635218 11.097819,15.5792643 C11.0418653,15.5233106 11.0138889,15.4553679 11.0138889,15.375434 L11.0138889,14.4162326 C11.0138889,14.3362988 11.0418653,14.268356 11.097819,14.2124023 C11.1537727,14.1564486 11.2217155,14.1284722 11.3016493,14.1284722 L12.2608507,14.1284722 C12.3407845,14.1284722 12.4087273,14.1564486 12.464681,14.2124023 C12.5206347,14.268356 12.5486111,14.3362988 12.5486111,14.4162326 L12.5486111,15.375434 Z M14.3710938,15.6631944 C14.2911599,15.6631944 14.2232171,15.635218 14.1672635,15.5792643 C14.1113098,15.5233106 14.0833333,15.4553679 14.0833333,15.375434 L14.0833333,14.4162326 C14.0833333,14.3362988 14.1113098,14.268356 14.1672635,14.2124023 C14.2232171,14.1564486 14.2911599,14.1284722 14.3710938,14.1284722 L15.3302951,14.1284722 C15.410229,14.1284722 15.4781717,14.1564486 15.5341254,14.2124023 C15.5900791,14.268356 15.6180556,14.3362988 15.6180556,14.4162326 L15.6180556,15.375434 C15.6180556,15.4553679 15.5900791,15.5233106 15.5341254,15.5792643 C15.4781717,15.635218 15.410229,15.6631944 15.3302951,15.6631944 L14.3710938,15.6631944 Z M18.6875,18.6367188 L18.6875,19.5 L7.94444444,19.5 L7.94444444,18.6367188 C7.94444444,18.5567849 7.97242087,18.4888421 8.02837457,18.4328885 C8.08432826,18.3769348 8.15227101,18.3489583 8.23220486,18.3489583 L8.71180556,18.3489583 L8.71180556,7.79774306 C8.69581879,7.63787536 8.74777501,7.50198985 8.86767578,7.39008247 C8.98757655,7.27817508 9.12745869,7.22222222 9.28732639,7.22222222 L17.3685981,7.22222222 C17.512479,7.22222222 17.6443679,7.27817508 17.7642687,7.39008247 C17.8841694,7.50198985 17.9361257,7.63787536 17.9201389,7.79774306 L17.9201389,18.3489583 L18.3997396,18.3489583 C18.4796734,18.3489583 18.5476162,18.3769348 18.6035699,18.4328885 C18.6595236,18.4888421 18.6875,18.5567849 18.6875,18.6367188 Z M9.86284722,18.3249783 L12.5486111,18.3249783 L12.5486111,16.718316 C12.5486111,16.6383821 12.5765875,16.5704394 12.6325412,16.5144857 C12.6884949,16.458532 12.7564377,16.4305556 12.8363715,16.4305556 L13.7955729,16.4305556 C13.8755068,16.4305556 13.9434495,16.458532 13.9994032,16.5144857 C14.0553569,16.5704394 14.0833333,16.6383821 14.0833333,16.718316 L14.0833333,18.3249783 L16.7930773,18.3249783 L16.7930773,8.39724392 L9.86284722,8.37326389 L9.86284722,18.3249783 Z"
              fill="#697C95"
              id="icon-building"
            />
          </g>
        </g>
      </g>
    </g>
  </g>
</svg>
`;
